sdk/python/feast/infra/feature_servers/aws_lambda/Dockerfile

```diff
@@ -9,8 +9,8 @@ COPY protos protos
 COPY README.md README.md
 
 # Install Feast for AWS with Lambda dependencies
-RUN pip3 install -e 'sdk/python[aws,redis]'
-RUN pip3 install -r sdk/python/feast/infra/feature_servers/aws_lambda/requirements.txt --target "${LAMBDA_TASK_ROOT}"
+RUN pip3 install --default-timeout=120 -e 'sdk/python[aws,redis]'
```

Comment:
any reason why this is increased from the default 15s?

Reply:
When building Docker image locally, the build kept timing out due to my network so I switched to 120s. I plan on removing this.